src/compresswindow.h

```diff
@@ -3,13 +3,13 @@
 
 #include "huffmanencoding.h"
 
-#include <QMainWindow>
+#include <QDialog>
 
 namespace Ui {
 class CompressWindow;
 }
 
-class CompressWindow : public QMainWindow
+class CompressWindow : public QDialog
```

Comment:
This is too, I'm need your input why changes into dialog.

Reply:
I've never seen a Qt Applicaion with more the one QMainWindow. This object is acting as a Dialog, so I've turned it into one.

Reply:
Ok. I'm quite understand. Can I have a button minimize, maximize in dialog? Previously I turn into MainWindow, because of this.

Reply:
> Ok. I'm quite understand. Can I have a button minimize, maximize in dialog? Previously I turn into MainWindow, because of this.
Yes This is how it looks as a Dialog.
![2019-10-01_Frog Archiver - Extract_1](https://user-images.githubusercontent.com/7450820/66010611-b170eb80-e48d-11e9-8bdb-adff162ee3b6.png)